docs/map.json

```diff
@@ -39,9 +39,9 @@
                   "file": "shared/tutorials/npm-workspaces"
                 },
                 {
-                  "name": "Integrated Monorepo",
```

Comment:
similarly I would move it to the end of the list, it's not the most important one